README.md

```diff
@@ -1,58 +1,64 @@
-# Breadcrumb
-Breadcrumb is a canonical component that any VTEX app can use.
+# VTEX Breadcrumb
+
+## Description
+The VTEX BreadCrumb is an app that shows a hierarchy of the current page in relation to the store structure and it is used by the Dreamstore product.
```

Comment:
What do you think of the description this way?

`The VTEX BreadCrumb is a secondary navigation scheme that reveals the user location on the store and it is used by the Dreamstore product.`